app/views/RoomView/index.js

```diff
@@ -452,6 +455,10 @@ class RoomView extends React.Component {
 		}
 	}
 
+	setFileLoading = (fileLoading) => {
```

Comment:
`setFileLoading = fileLoading => this.setState({ fileLoading });` ?

Reply:
if I do it this way I get eslint:  error    `Unexpected parentheses around single function argument having a body with no curly braces  arrow-parens`